src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -224,6 +224,26 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         }
     }
 
+    void TermControl::SearchNextMatch()
+    {
+        if (!_searchBox)

```

Comment:
@Hegunumo, @zadjii-msft  - product question: if the search box is collapsed and we invoke "search next" binding, shouldn't we open a search box and try to search with the last needle (if exists)?

Reply:
Okay, so I maybe mentioned this in #8588 - Maybe we should! Or we don't necessarily need to open the search dialog, but we could just move to the next needle. 

It's a good idea, and something we should probably stick in #3920 as a follow up to (both these PRs)